content/people/dcallega.md

```diff
@@ -1,93 +1,27 @@
 ---
-# Page front matter
-#
-# The text between the '---' lines at the top of this file defines the
-# metadata, which is used to provide a unified profile at the top of
-# this personal page and in the people directory page.
-#
-# The metadata are formatted in YAML, with a small subset of its
-# syntax used. Format your information the same way it is done
-# for the exemplary data below.
-#
-# Do not copy the lines starting with '#' (like this one) into your
-# file; those are just comments.
-#
-# Full name, e.g., 'Firstname Lastname' or 'Firstname M. Lastname'.
-# Write it the same way you would in a paper.
 name: Davide Callegaro
-# Formal status, e.g., 'student-phd' or 'student-masters'.
-# (See people.roles in ../config.yaml for the full list of roles.)
 role: student-phd
-# Free-form status in your studies, e.g., 'Ph.D. Student',
-# 'Ph.D. Candidate' (after candidacy),
-# 'Master’s Student (co-advised Prof. Example)', etc.
 status: Ph.D. Student
-# Year when joined the group.
 yearjoined: 2016
-# Filename of an optional portrait photo to be featured on the page.
-# The file should be put in the ../static/img/people/ directory.
 portrait: dcallega.jpg
-# Filename of an optional avatar photo to be shown in the people
-# directory page. Should be square. If missing, the portrait photo is
-# used instead. The file should be put in the same directory as the
-# portrait photo.
 badge: dcallega-badge.jpg
-# A list of free-form topics constituting your main research
-# interests. Keep each short and to the point; do not put a period
-# at the end.
 interests:
   - Distributed Machine Learning in Heterogeneous Computer Networks
   - Reinforcement Learning and Semantic Maps
   - Unmanned Aerial Vehicles
   - Edge Computing
-# A list of acquired degrees, in reversed chronological order,
-# each described by three fields: official degree title, name of
-# the university, and year of graduation.
 education:
   - degree: M.S. in Computer Engineering
     institution: Università degli Studi di Padova
     year: 2016
   - degree: B.S. in Information Engineering
     institution: Università degli Studi di Padova
     year: 2013
-# Optional email address.
 email: dcallega@uci.edu
-# Optional URL of your personal website (outside of the lab website).
-# Do not forget to include the protocol part, e.g., 'http://' or 'https://'.
 website: https://www.linkedin.com/in/davide-callegaro/
```

Comment:
Website entry syntax [has changed][commit] to support (multiple) links with
titles. Please change this to:

    websites:
      - title: LinkedIn
        url: https://www.linkedin.com/in/davide-callegaro/

(In general, the `title` field is optional.)

[commit]: https://github.com/uci-iasl/website/commit/46182f701b9a2d85ced05103826c242ea412af3c

Reply:
done